Homework-5/task_4.py

```diff
@@ -0,0 +1,14 @@
+n = int(input("Enter a number (0 < n < 50): "))
+if n <= 0 or n > 50:
+    print("Please enter a number between 1 and 50")
+else:
+    row_count = 1
+
+    print(f"{" " * n}*")
```

Comment:
f''-ში არასწორად ფორმატიების გამო ერორები გაქვს. გთავაზობ ალტერნატივას:
n = int(input("Enter a number (0 < n < 50): "))
if n <= 0 or n > 50:
    print("Please enter a number between 1 and 50")
else:
    row_count = 1
    print(" " * n + "*")
    while row_count <= n:
        if row_count != n:
            print(" " * (n - row_count) + "/" * row_count + "|" + "\\" * row_count)
            row_count += 1
        else:
            print(" " * n + "|")
            break

Reply:
რა განსხვავებაა ამათ შორის?

თქვენ მიერ მოცემულ კოდში:
`row_count = 1`
`print(" " * n + "*")`
და ჩემსაში
`row_count = 1`
`print(f"{" " * n}*")`

ორივე ვარიანტი white space ხატავს n-ჯერ და ამატებს ფიფქს
output-შიც განსხვავებას ვერ ვხედავ ჩემს კოდსა და ამაში, ერორი არის ანუ execution ვერ ხორციელდება?
დანარჩენი კოდიც ზუსტად იგივეა, ცოტა დავიბენი 😕